test/performance/kueue/config.yaml

```diff
@@ -0,0 +1,175 @@
+{{$MODE := DefaultParam .MODE "Indexed"}}
```

Comment:
Is the only difference that this file creates a LocalQueue?
I suppose there is no way to do inheritance in cluster loader, is there?

But maybe you can embed? We should avoid having so much duplicated configuration.

Reply:
Parametrized the `config.yaml` to support with and without kueue scenarios:
https://github.com/kubernetes-sigs/kueue/pull/462/commits/febfe07584a73e03832dff42685d67e396bf7c1d